user_guide_src/source/incoming/routing.rst

```diff
@@ -544,7 +546,16 @@ It is possible to nest groups within groups for finer organization if you need i
 
 This would handle the URL at **admin/users/list**.
 
-.. note:: Options passed to the outer ``group()`` (for example ``namespace`` and ``filter``) are not merged with the inner ``group()`` options.
+**Filter** option passed to the outer ``group()`` are merged with the inner
+``group()`` filter option.
+The above code runs ``myfilter:config`` for the route ``admin``, and ``myfilter:config``
+and ``myfilter:region`` for the route ``admin/users/list``.
+
+But other options that you specify in the inner ``group()`` options, the value
+is overwritten.
```

Comment:
```suggestion
Any other overlapping options passed to the inner `group()` will overwrite their values.
```

Reply:
Eh this is tricky to word, I'm not sure mine is strictly better. Take it or leave it.